### DISCUSSION TITLE
Finally Migrated! But have some problems...

### DISCUSSION THREAD

**Discussion by dogzipp** [nzbgetcom/nzbget#235] [General]
Finally migrated from old v21 to new v23. Also used a Synocommunity package to install and carefully had to migrate my scripts, and tinker some with permissions, and other things to make it run successfully.

But I am having two problems... One I get this:

nzbget.conf(0): Invalid value for option "Task0.Time": "*:50 ##"

I checked the configuration file on /var and there's no Task0.Time entry at all, so not sure where it's coming from.

Also my scheduler scripts, seem to return this error (worked fine under v21)

Completion: SyntaxError: invalid syntax
Completion: ^
Completion: except Exception, e:

Python version is the same as before (3.8).

Any help is appreciated, since I am trying new things, before I go and help my friends upgrade their Synology NASes as well.

**Comment by dnzbk** [comment]
Unfortunately, it's hard to understand what the problem might be right now. 
The first thing that comes to mind is to double check the `Completion` extension settings and resave `(SaveAllChanges)` them.
Where did you get this extension? Сould you share a link? 
I'd like to test it, thanks.

**Comment by dogzipp** [comment]
Managed to fix the completion problems, seems it's not entirely compatible.with Python 3, so I just changed the first line of the script and pointed it to Python 2 (got it from the nzbget.net download page a long time ago). There a Python 3 update from somebody else, but I just got a different error, so I decided to keep the one that worked for me.

https://forum.nzbget.net/viewtopic.php?f=8&t=1736

As for the Task0, I never could get rid of thta message, so I believe it's a v23 bug. I checked all the conf file, and not one has such an entry, so I downgraded back to v21 and it went away.

I've been using NZBget for more than 12+ years and I love that somebody is back in the driver seat of Nzbget, but some things are being done in a way that Hugbug would never have done it, so they clash with the rest of the interface (for example, the new extension manager and all the colored buttons).

**Reply by luckedea** [reply]
Completion is an old Python2 extension which needs a big revamp. Tried looking into it before, but it needs more attention, updates and testing, that's coming up soon.

I understand what you mean about Hugbug's approach and vision, but we are also trying to make the app easier to live with - current problems of extensions required a new approach. Extensions Manager is not ideal, it will get improvements and more extensions supported. Extensions Manager should not break old extensions though!

Keeping NZBGet lean, lightweight, flexible and working correctly on all of the environments is not an easy task already, so EM is the way to move forward with all the extensions, new python versions and all related libs.

Any feedback provided at this point is very valuable, thank you. And any related bugs we are being looked at in high priority :)

**Reply by dogzipp** [reply]
Thanks for your reply. I you read my reply carefully, I am not bashing the improvements (I am actually in favor of the Extension Manager), mostly I am talking is about the interface/implementation. NZBget has allways had a pretty elegant and clean interface, and the Extension Manager just introduced a lots of colors and icons that's pretty out of character in the rest of the interface.

I am eager to try the next version that comes out, hopefully mu Task0 bug get's ironed out. (I also noticed that Scheduled Tasks repeated launched like 3-4 instead of just once)

**Reply by luckedea** [reply]
@dogzipp You brought up a good point, so I thought a "bigger" more generic response would make sense. We are trying to balance between old and new, I really value opinion of old-time users.

For example, I am personally not a fan of dark theme of the UI, but hey - it's being released asap, too many users want it.

We would get more fixes and improvements in, and in 1-2 releases we'll polish the UI. Current framework doesn't allow us to do both - if we change the "core", it isn't pretty on the UI :)

**Reply by dogzipp** [reply]
I understand. I've tried to make a "dark" theme on DOGnzb for a while now, and I am never a fan of how it looks. Never the less, I would probably need to release it a some point lol.
